nevergrad/functions/arcoating/test_core.py

```diff
@@ -34,7 +34,7 @@ def test_arcoating_recombination() -> None:
         arrays.append(func.parametrization.spawn_child())  # type: ignore
         arrays[-1].value = num * np.ones(arrays[0].value.shape)
     arrays[0].recombine(arrays[1])
-    expected = [[3., 3., 5., 5., 5., 3.]]
+    expected = [3., 3., 3., 5., 3., 3.]
```

Comment:
can you tell me a bit more how this expected is built ? this is a test of seedability ?

Reply:
Exactly, this both:
- tests that it is seedable (seed is set to 24 a few lines above)
- that we obtain something expected (here all the 3 come from the first array, and the 5 from the second), so we indeed have a 2 points crossover.

The value is different from before because I changed the implementation.